apps/server/cypress/e2e/suite1/testExample.cy.ts

```diff
@@ -38,6 +40,96 @@ const makeABConnection = async (jobType) => {
   expectConnectionSuccess();
 };
 
+const getAccountId = ({ aggregator, memberGuid, userId }) => {
+  const url = `/data/aggregator/${aggregator}/user/${userId}/connection/${memberGuid}/accounts`;
+
+  return cy.request("get", `/api${url}`).then((dataResponse) => {
+    const accountId = dataResponse.body.accounts.find(
+      (acc) => Object.keys(acc)[0] === "depositAccount",
+    ).depositAccount.accountId;
+
+    return accountId;
+  });
+};
+
+const verifyTransactionsValidatorSuccess = ({
+  accountId,
+  aggregator,
+  userId,
+}) => {
+  const url = `/data/aggregator/${aggregator}/user/${userId}/account/${accountId}/transactions${aggregator === TEST_EXAMPLE_B_AGGREGATOR_STRING ? "?start_time=2021/1/1" : ""}`;
+
+  return cy
+    .request({
+      method: "GET",
+      url: `/api${url}`,
+    })
+    .then((dataResponse) => {
+      expect(dataResponse.status).to.equal(200);
+      expect(dataResponse.body.transactions.length).to.be.greaterThan(-1);
+    });
+};
+
+const verifyTransactionsValidatorError = ({
+  accountId,
+  aggregator,
+  userId,
+}) => {
+  const url = `/data/aggregator/${aggregator}/user/${userId}/account/${accountId}/transactions`;
+
+  return cy
+    .request({
+      method: "GET",
+      url: `/api${url}`,
+      failOnStatusCode: false,
+    })
+    .then((dataResponse) => {
+      expect(dataResponse.status).to.equal(400);
+    });
+};
+
+export const generateValidatorTests = ({ makeAConnection }) => {
```

Comment:
The generateValidatorTests function feels like a premature abstraction. It's only used once. I'd just put the it statement directly in the describe